docs/sources/next/results-output/real-time/opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+
+This is an extension, and we are looking for feedback and contributions and aiming to graduate it as an experimental output within k6 core in one of the future releases. Feel free to open an issue directly in the [grafana/xk6-output-opentelemetry repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
+
+## Metrics mapping
+
+All [k6 metric types](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/metrics) are converted into an equivalent [OpenTelemetry metric type](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#timeseries-model). The k6's tags converted to the OpenTelemetry attributes. Whenever it's possible, the units also passed with the metrics.
+
+| k6      | OpenTelemetry                                                                                                                                                                                                                                                                                                                                                                                      |
+| ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Counter | `Float64CounterOption`                                                                                                                                                                                                                                                                                                                                                                             |
+| Gauge   | `Float64ObservableGauge`                                                                                                                                                                                                                                                                                                                                                                           |
+| Rate    | It splits into two `Int64Counter` counters with the names `metric_name.occurred` and `metric_name.total`. The `metric_name.occurred` counts only the number of non-zero occurencies, where the `metric_name.total` registers total number of positive and negative. We might reconsider it after the [xk6-output-opentelemetry#12](https://github.com/grafana/xk6-output-opentelemetry/issues/12). |
```

Comment:
```suggestion
| Rate    | Split into two `Int64Counter` counters named `metric_name.occurred` and `metric_name.total`. `metric_name.occurred` counts only the number of non-zero occurrences, and `metric_name.total` registers the total number of positive and negative occurrences. This might change in the future, refer to [xk6-output-opentelemetry#12](https://github.com/grafana/xk6-output-opentelemetry/issues/12) for more details. |
```